README.md

```diff
@@ -28,30 +28,30 @@ yarn add @shopify/polaris
 
 ### Usage
 
-1.  Include the CSS in your HTML:
+1.  Import the CSS directly into your project if your asset packager supports it:
+
+```js
+import '@shopify/polaris/styles.css';
+```
+
+Otherwise include the CSS in your HTML. We suggest copying the styles file into your own project, but you may also use it directly:
```

Comment:
I'm thinking more along the lines that generally don't want to add an extra point of failure your app - why pull in stuff from another host when you can your own.

That tradeoff affects all 3rd party code it doesn't seem that polaris specific so i left it out as it felt like already implied knowledge.

How about: `We suggest copying the styles file into your own project to avoid adding extra hosts you depend upon, but you may also use it directly`?

Reply:
I guess I just find it confusing because using a CDN is supposed to be better performance wise than having a bunch of styles local to your app. Having all of the Bootstrap styles local to your app for instance would be insane vs having the CDN link. I'm struggling to find it convincing to consumers as something that would be a useful tradeoff. 

> extra point of failure your app

If we frame copying it to your app locally as a fallback in case the CDN is down does that makes sense? Or are you thinking of other points of failure?

Reply:
This article is handy https://csswizardry.com/2019/05/self-host-your-static-assets/

> a CDN is supposed to be better performance wise than having a bunch of styles local to your app.

Broadly speaking yes. However much of those benefits are lost if you you've got one asset on a different host to everything else - it's another host you have to negotiate a connection with and you lose any resource prioritization. The time you lose to those things offset any gain you'd get from using a CDN. It's preferable to keep all your critical static assets on a single host - even if that host isn't fronted by a CDN.

> Having all of the Bootstrap styles local to your app for instance would be insane vs having the CDN link. 

Indeed it would, however having the single `boostrap.min.css` file hosted on the same server as your other static assets would be preferable to both, which is the style of hosting we're talking about :)

> If we frame copying it to your app locally as a fallback in case the CDN is down does that makes sense?

I think that pattern adds extra complication for negligible additional resilience. I don't think it is worth explaining or promoting it.

Reply:
> The time you lose to those things offset any gain you'd get from using a CDN. It's preferable to keep all your critical static assets on a single host - even if that host isn't fronted by a CDN.

Ahhh okay, that makes sense. Thank you for clarifying!

Reply:
Any time :)